### PR TITLE
Added i18n support to SEO meta tags

### LINKED ISSUE #63: i18n Metatags
**Context:**
Note this should be after we merge #57.
Whenever sharing links of the website, `/fr` or `/es` it does not show the translation of the title or description of the page in that native language. The goal would be to fix that with an existing library.

A conversation was had on potentially using `next/head` instead of `react-helmet`, as a reference.
https://stackoverflow.com/questions/61063193/react-helmet-or-next-head-for-next-js-project#61063491

**Current State:**
When debugging the french or spanish website with Facebook's sharing debugger, the following comes up:
[Link](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fdeveloper-dao-lewqo5jyg-developdao.vercel.app%2Ffr) Note: link may expire, see screenshot as alternative.

Screen Shot 2021-09-14 at 6 38 27 PM

**Proposal:**
Implement a library that works with i18n and make sure it works with Facebook's sharing debugger.

### PR DESCRIPTION
**Pull request by fmoliveira**
Closes #63 

- Moved SEO title and description meta tags to translation files
- Removed duplicate description meta tag
- Added Spanish translation for the page description (proofread wanted ⚠️ )
- Changed social banner meta tag for absolute URL so it can be displayed properly on Twitter

Preview on Facebook Debugger:
Screen Shot 2021-09-19 at 12 14 40 AM

Preview on Twitter Card Validator:
Screen Shot 2021-09-19 at 12 15 24 AM

### CODE REVIEW

File: public/locales/es/common.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "title": "Developer DAO",
+  "description": "Una comunidad de constructores que creen en la propiedad colectiva de Internet.",
```
Comment (fmoliveira) [inline public/locales/es/common.json:3]: 🇪🇸  Proofreader wanted, I added this so we could test on the meta tags validator cc @gjsyme @codingwithmanny

File: src/pages/_app.js [hunk 2]
```diff
@@ -2,46 +2,40 @@ import React from 'react';
...
 import { ChakraProvider } from '@chakra-ui/react';
-import { theme } from '../theme';
+import { useTranslation } from 'next-i18next';
 import '@fontsource/inter/variable-full.css';
 import '@fontsource/source-code-pro/400.css';
 
+import { DEVELOPER_DAO_WEBSITE } from '../utils/DeveloperDaoConstants';
+import { theme } from '../theme';
+
+const socialBanner = `${DEVELOPER_DAO_WEBSITE}/social-banner.png`;
```
Comment (fmoliveira) [inline src/pages/_app.js:12]: [Twitter Card Validator](https://cards-dev.twitter.com/validator) doesn't show social images with relative URLs.

Screen Shot 2021-09-19 at 12 21 05 AM

Comment (codingwithmanny) [reply src/pages/_app.js:12]: Nice. I completely forgot about the twitter card validator.
I'm almost thinking that this is something we might want to include as a default thing to test for down the road for different pages. Nice work @fmoliveira

**Review by codingwithmanny — APPROVED** [review]
LGTM

**Review by with-heart — APPROVED** [review]
Love it!

**Comment by fmoliveira** [comment]
Thank you so much @codingwithmanny and @with-heart !

Feel free to merge it at any time. 🎉

File: src/pages/_app.js (hunk 2 above)
Comment (fmoliveira) [reply src/pages/_app.js:12]: Thank you so much @codingwithmanny ! :)

I agree with testing this continuously. Customizing the title and description on a page basis would provide more context on links shared on social networks, and may slightly boost SEO.

💡  Random ideas
Customizing OpenGraph images for certain pages could be even cooler. In the future, sharing the URL to a specific token could use that NFT's pixel avatar as the OG image.

**Comment by with-heart** [comment]
Rebased with `main`

**Pull request merged**